createInvoice/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.mvc.view.suffix= .html
+
+logging.level.org.springframework.web=INFO
+logging.level.guru.springframework.blogs.controllers=INFO
+logging.level.org.hibernate=ERROR
+logging.file=logs/spring-boot-logging.log
+log4j.logger.org.thymeleaf=DEBUG
+
+spring.thymeleaf.cache=false
+
+
+cloudelements.url=https://api.cloud-elements.co.uk
```

Comment:
It might be better to default this to staging.

Reply:
removed it. I am probably the biggest user of this ( :) ) and I mainly use prod. Plus, the readme describes you need to add these properties or use the /environment path to set the vars, including the URL